src/jit/morph.cpp

```diff
@@ -2069,11 +2069,9 @@ GenTreePtr    Compiler::fgMakeTmpArgNode(unsigned tmpVarNum
             if (structSize <= MAX_PASS_MULTIREG_BYTES)
             {
                 assert(structSize > TARGET_POINTER_SIZE);  // structSize must be 9..16
-
-                // ToDo-ARM64: Consider using:  arg->ChangeOper(GT_LCL_FLD);
-                // as that is how FEATURE_UNIX_AMD64_STRUCT_PASSING works.
-            
-                // Pass by value in two registers
+                                                           // ToDo-ARM64: Consider using:  arg->ChangeOper(GT_LCL_FLD);
+                                                           // as that is how FEATURE_UNIX_AMD64_STRUCT_PASSING works.
+                                                           // Pass by value in two registers
```

Comment:
No - that was my new install of VS - I had forgotten to reset all the auto-format stuff. I thought I'd fixed all the things it changed, but apparently missed this one. I'll fix it.